react-frontend/src/components/listRecognitions.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import { Row, Col, Label, Button, Container } from "reactstrap";
+import "bootstrap/dist/css/bootstrap.min.css";
```

Comment:
Please remove this line.

1. We already have imported bootstrap CSS at a common place.
2. Every component should not include bootstrap CSS